provider/infoblox/infoblox.go

```diff
@@ -636,6 +744,45 @@ func (p *ProviderConfig) createRecords(created infobloxChangeMap) {
 					)
 					continue
 				}
+				expTypeA := ibclient.NewEmptyRecordA().ObjectType()
+				expTypeTXT := ibclient.NewEmptyRecordTXT().ObjectType()
+				expTypePTR := ibclient.NewEmptyRecordPTR().ObjectType()
+				expTypeCNAME := ibclient.NewEmptyRecordCNAME().ObjectType()
```

Comment:
It doesn't make much sense to create those variable for each `for` loop iteration. I think that they are better to be defined in `const` section at the very top of the file.

Reply:
ok

Reply:
Yes true, they are computed at runtime, so they cannot be constants. `var` section would be ok too, my thought was to move avoid their creation at each iteration.